migrations/2023-08-11-112158_auto_resolve_report/up.sql

```diff
@@ -0,0 +1,51 @@
+-- Automatically resolve all reports for a given post once it is marked as removed
+-- TODO: how to set `resolver_id`?
```

Comment:
Instead of using the `post` table as the triggering one, use an insert on the  `mod_remove_post` table, and `mod_remove_comment` tables. A few benefits:

- It has the `mod_person_id`, that you can use to set the `resolver_id` .
- It has a `removed` column, meaning you could *resolve* or *unresolve* reports based on an insert to that table.

Reply:
Right that makes sense, changed it. Though Im not unresolving reports, it seems unnecessary.